pulpcore/tests/functional/api/test_crud_users.py

```diff
@@ -25,6 +25,7 @@ def setUp(self):
         """Create an API client."""
         self.client = api.Client(self.cfg, api.json_handler)
 
+    @unittest.skip("User creation via API is not allowed until Pulp has proper user management")
```

Comment:
We can skip the whole class, instead of individual methods. Then we will have just one change to adjust later. It is fine as it is, just a suggestion.

Reply:
@bmbouter , if we remove them and not just skip them, they will be definitely forgotten :(, in my opinion.

Reply:
Agree. While I like version control, `out of sight, out of mind` is still a thing.

+1 to not deleting the tests and then accidentally re-writing them later.

The removal of create and update remove feature parity when compared to Pulp 2, correct?

If so, wouldn't feature parity be on the list for the RC? 

My apologies if I am off-base.

Reply:
@bherrin3 , there will be no Pulp 2 feature parity in Pulp 3.0 release, expect it in later Y releases and only for the features we decided not to drop.

Reply:
I totally see what you are saying if we are bringing back user management. I was convinced from several discussion w/ devs that Pulp3 can't offer Users of any kind for probably a long time, possibly years. This PR isn't the best place to bring up that concern so I think I'll need to mail pulp-dev to see what the community thinks.

To share knoelwedge on the pulp2/pulp3 feature parity, the feature set of Pulp3 has many things missing from Pulp2. Some were conspicuously left behind, others just aren't a priority based on feedback from the community and various stakeholders. In many ways the features themselves are so different it's hard to even match them up. So for the RC the only remaining items are those on the ever-changing RC blockers list here:  https://pulp.plan.io/issues?query_id=121 This is my understanding at least, but others could see it differently.

Reply:
For clarity, I'm lgtm on this as-is while we figure out the future of Users some other time. Thanks for putting this together!

Reply:
I could see user management not coming back at all (at least not in its current form) so +1 to removing these tests from me.